### PR TITLE
[fix](memory) Print all query/load memory before memory GC when `memory_debug=true`

### PR DESCRIPTION
**Pull request by xinyiZzz**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Print the mem tracker list of all query/load

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by xinyiZzz** [comment]
run buildall

File: be/src/util/mem_info.cpp [hunk 1]
```diff
@@ -179,18 +184,30 @@ bool MemInfo::process_full_gc() {
         }
     }
 
+    if (doris::config::memory_debug) {
+        LOG(INFO) << MemTrackerLimiter::type_detail_usage("Before free top memory query in Full GC",
```
[the remark this answers is not in the document]
Comment (xinyiZzz) [reply be/src/util/mem_info.cpp:188]: 可是Doris没有 LOG(DEBUG) 啊。。只有 INFO WARNING ERROR FATALDFATAL
要加一个么？

[the remark this answers is not in the document]
Comment (xinyiZzz) [reply be/src/util/mem_info.cpp]: done，改成了VLOG

**Comment by xinyiZzz** [comment]
run buildall

**Comment by xinyiZzz** [comment]
run buildall

**Comment by xinyiZzz** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**